app/less/default/core/scaffolding.less

```diff
@@ -0,0 +1,18 @@
+//
+// Scaffolding
+// --------------------------------------------------
+
```

Comment:
would suggest putting this in a file call `scrollbar.less` rather then using a generic file, which will end up having lots of random styles for various elements inside.